Mage/src/main/java/mage/abilities/Modes.java

```diff
@@ -437,8 +456,19 @@ public void addSelectedMode(UUID modeId) {
             throw new IllegalArgumentException("Unknown modeId to select");
         }
 
+        Mode mode = get(modeId);
+
+        if (mode.getPawPrintValue() < 0 && this.getMaxPawPrints() >= 0){
+            throw new RuntimeException("Selected mode has no pawprint value, but this Modes object requires pawprint modes!");
+        }
+        if (mode.getPawPrintValue() >= 0 && this.getMaxPawPrints() < 0) {
+            throw new RuntimeException("Selected mode has a pawprint value, but this Modes object has no pawprints!");
+        }
+
+        selectedPawPrints += mode.pawPrintValue;
+
         if (selectedModes.contains(modeId) && mayChooseSameModeMoreThanOnce) {
-            Mode duplicateMode = get(modeId).copy();
+            Mode duplicateMode = mode.copy();
```

Comment:
what's up with this change?

Reply:
I'm using the Mode obtained from `get(modeId)` multiple times so i bumped it to a local variable